code/game/objects/items/trash.dm

```diff
@@ -94,6 +98,10 @@
 	name = "snack bowl"
 	icon_state	= "snack_bowl"
 
+/obj/item/trash/small_bowl
+	name = "small bowl"
+	icon_state	= "small_bowl"
+

```

Comment:
Love the sprites, but this (and the bowl above it which was an oversight) should probably use
drop_sound = 'sound/items/drop/food.ogg'
pickup_sound = 'sound/items/pickup/food.ogg'